### PR TITLE
pass IMAGE_* variables to e2e commands

### PR DESCRIPTION
**Pull request by MatousJobanek**
pass IMAGE_* variables to e2e commands so we can use them to deploy the images to cluster.

### CODE REVIEW

File: ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml [hunk 1]
```diff
@@ -53,6 +53,6 @@ tests:
   container:
     from: src
 - as: e2e
-  commands: make test-e2e
+  commands: IMAGE_HOST_OPERATOR=${IMAGE_HOST_OPERATOR} IMAGE_MEMBER_OPERATOR=${IMAGE_MEMBER_OPERATOR} IMAGE_REGISTRATION_SERVICE=${IMAGE_REGISTRATION_SERVICE} make test-e2e
```
Comment (droslean) [inline ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml:56]: the value here is repeated.

File: ci-operator/jobs/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master-presubmits.yaml [hunk 2]
```diff
@@ -92,7 +92,8 @@ presubmits:
         - name: JOB_NAME_SAFE
           value: e2e
         - name: TEST_COMMAND
-          value: make test-e2e
+          value: IMAGE_HOST_OPERATOR=${IMAGE_HOST_OPERATOR} IMAGE_MEMBER_OPERATOR=${IMAGE_MEMBER_OPERATOR}
+            IMAGE_REGISTRATION_SERVICE=${IMAGE_REGISTRATION_SERVICE} make test-e2e
```
Comment (droslean) [inline ci-operator/jobs/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master-presubmits.yaml:96]: same here.

File: ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml (hunk 1 above)
Comment (MatousJobanek) [reply ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml:56]: I don't understand - this is a different job that the other ones - I need to build all three images to make this job running.

**Pull request merged**